conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosNodeOperator.scala

```diff
@@ -338,24 +338,60 @@ private[tezos] class TezosNodeOperator(
       (fetchCurrentQuorum, fetchCurrentProposal).mapN(CurrentVotes.apply)
     }
 
-  /** Fetches proposals for given blocks */
+  /** Fetches proposals for given blocks
+    * The returned value, when available for active voting periods, is the ID of
+    * a proposal, though for the sake of naming consistency with tezos schema
+    * we refer to that as a protocol ID.
+    */
```

Comment:
The word 'proposals' is ambiguous. It could mean proposal operations, suggested proposal hashes or active proposals. Therefore, I recommend renaming this function to `getActiveProposals`.